opennms-doc/guide-admin/src/asciidoc/text/telemetryd/protocols/ipfix.adoc

```diff
@@ -25,8 +24,8 @@ The IPFIX UDP Parser supports protocol detection.
 [options="header, autowidth"]
 |===
 | Parameter             | Description                                                                    | Required | Default value
-| `templateTimeout`     | Templates must be re-declared in the given duration or they will be dropped    | no       | 30 minutes
-| `maxClockSkew`        | The maximum delta in seconds between exporter and Minion timestamps.       | no       | 0
+| `templateTimeout`     | Templates must be redeclared in the given duration or they will be dropped.    | no       | 30 minutes
+| `maxClockSkew`        | The maximum delta in seconds between exporter and Minion time stamps.       | no       | 0
```

Comment:
Isn't "timestamp" a single word as it's used as a technical term?

Reply:
I prefer one word, and that's what the Oxford dictionary specifies, but Webster's has two words. (I've been trying to use one dictionary as a standard, and chose Webster's because it is American.) I'll go back to one word, though, as I see that's what Wikipedia uses and it seems to be industry standard. Thanks for pointing this out. 
Now, if we can persuade the Americans to use Canadian/British spelling I can go back to including the letter "u" to words like favourite, labour, arbour, honour, etc. :-)